src/App.js

```diff
@@ -25,7 +26,7 @@ function App() {
     return (
         <div className="w-screen h-screen flex flex-col justify-between items-center">
             <header className="flex bg-amber-500 text-white w-3/12 h-20 items-center text-center px-8 mt-4 rounded-3xl z-30">
-                <h2 className="text-sm mr-4">Dele Dele</h2>
+                <img src={deleDele} alt="logo" className="w-10 h-10 mr-4"/>
```

Comment:
Can you resize de image logo?

Reply:
adjust the size of the logo with the required changes